hexrd/nf_config/experiment.py

```diff
@@ -33,33 +33,38 @@ def max_tth(self):
     def comp_thresh(self):
         key = 'experiment:comp_thresh'
```

Comment:
No need to set a variable for `key`, this can be removed - just refer to the string directly.